### PR TITLE
Fix: ObjectBrick with multiple Localized fields are broken after update to 6.9/10.0

### LINKED ISSUE #9646: ObjectBrick with multiple Localized fields are broken after update to 6.9/10.0
# Bug Report
If you have an object brick with multiple localized fields, its impossible to get the fields or save it in the backend.
For example:
image

image

image

This was all working in 6.7

### Expected behavior

Save and Read the fields

### Actual behavior

Getter doesn't exists and save in the admin interface doesn't work

### PR DESCRIPTION
**Pull request by blankse**
Resolve #9646

### CODE REVIEW

**Comment by blankse** [comment]
Classificationstore has also setReferencedFields/addReferencedField but no getFieldDefinitions method which call getReferencedFields. Is it there still needed?

**Review by dvesh3 — CHANGES_REQUESTED** [review]
since this needs to regenerate brick tables...please add migration to save objectbricks

File: bundles/CoreBundle/Migrations/Version20210706090823.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+use Pimcore\Model\DataObject;
+
+/**
+ * @internal
+ */
+final class Version20210706090823 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```
Comment (dvesh3) [inline bundles/CoreBundle/Migrations/Version20210706090823.php]:
```suggestion
        return 'Rebuild class definitions to fix Objectbricks with multiple Localized fields';
```

**Comment by dvesh3** [comment]
@blankse thanks a lot!

**Pull request merged**